Mail/Views/Alerts/LogoutConfirmationView.swift

```diff
@@ -43,12 +43,19 @@ struct LogoutConfirmationView: View {
         Task {
             @InjectService var notificationService: InfomaniakNotifications
             await notificationService.removeStoredTokenFor(userId: account.userId)
+
+            AccountManager.instance.removeTokenAndAccount(account: account)
+            if let nextAccount = AccountManager.instance.accounts.first {
+                AccountManager.instance.switchAccount(newAccount: nextAccount)
+            }
+            AccountManager.instance.saveAccounts()
         }
-        AccountManager.instance.removeTokenAndAccount(token: account.token)
-        if let nextAccount = AccountManager.instance.accounts.first {
-            AccountManager.instance.switchAccount(newAccount: nextAccount)
-        }
-        AccountManager.instance.saveAccounts()
+    }
+}
+
+extension ApiToken: Identifiable {
```

Comment:
Why ApiToken needs to be Identifiable here?

Reply:
It needs to be identifiable to be presented as a an item in the sheet before. I moved this extension in the view that this presenting the logout confirmation